### PR TITLE
Fix CRI-O configuration file location for newer versions

### PR DESCRIPTION
**Pull request by saschagrunert**
Fixes https://github.com/kata-containers/tests/issues/2400

### CODE REVIEW

File: .ci/install_crio.sh [hunk 1]
```diff
@@ -113,6 +112,16 @@ echo "Installing CRI Tools"
 crictl_url="${crictl_repo}/releases/download/v${crictl_version}/crictl-${crictl_tag_prefix}${crictl_version}-linux-$(${cidir}/kata-arch.sh -g).tar.gz"
 curl -Ls "$crictl_url" | sudo tar xfz - -C /usr/local/bin
 
+# Change CRI-O configuration options
+crio_config_file="/etc/crio/crio.conf"
+
+# This commit contains changes regarding CRI-O’s default configuration files,
+# so we have to use a new default config path from here.
+# https://github.com/cri-o/cri-o/commit/0f1226b99685f95e83c94dc6668b6452df5056db
+if git merge-base --is-ancestor 0f1226b99685f95e83c94dc6668b6452df5056db HEAD; then
```
Comment (grahamwhaley) [inline .ci/install_crio.sh]: nit: I wonder if we'd like to re-direct the git output to `/dev/null` or somewhere, for in the case where the patch fails, we will probably get a big 'git fail' error in the logs, which might be confusing when checking unrelated CI failures. Not a biggie though.

Comment (saschagrunert) [reply .ci/install_crio.sh]: Hm the git command does not output anything in success and error case :man_shrugging:

Comment (grahamwhaley) [reply .ci/install_crio.sh]: > Hm the git command does not output anything in success and error case 🤷‍♂

Oh, OK - cool. surprised about the error case ;-).

**Comment by jodh-intel** [comment]
Hi @saschagrunert - Travis is failing and if you look in the log it shows:

```
Found 1 commit between commit 42f9ee0d717661a6c43ddb90be234dc4494dbe2c and branch master
ERROR: Commit 42f9ee0d717661a6c43ddb90be234dc4494dbe2c: Failed to find subsystem in subject: "Fix CRI-O configuration file location for newer versions"
ERROR: checkcommits failed. See the document below for help on formatting
commits for the project.
https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format
```

Please can you tweak your commit format as explained in:

- https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format

**Comment by saschagrunert** [comment]
Thank you both for the hints, I updated my commit message.

**Pull request merged**